sdk/src/quotes.ts

```diff
@@ -312,7 +313,8 @@ export const getQuote = async (
  *                                                         stored destination address
  */
 export const getCrosschainQuote = async (
-  params: QuoteParams
+  params: QuoteParams,
+  abortSignal?: AbortSignal
```

Comment:
do we care about having this in the `getClaimBridgeQuote` (line 355) to pass along to `fetchAndSanityCheckCrosschainQuote` (line 386) as well?

Reply:
Great call, I was only thinking of the swaps flow, but there's no reason not to guard against stale quotes on claim bridges as well! ty